tests/test_model_index.args

```diff
@@ -0,0 +1,21 @@
+base_simple3.nit Obj --name-prefix --no-color
```

Comment:
you should use `test_prog` instead of `base_simpe3`  that is your toy program with many classes and modules (and that does not depend on core, not even core::kernel)